addOns/exportreport/exportreport.gradle.kts

```diff
@@ -10,9 +10,24 @@ zapAddOn {
     }
 }
 
+configurations {
+    "implementation" {
+        // Not needed.
+        exclude(group = "org.apache.odftoolkit", module = "taglets")
+    }
+}
+
 dependencies {
     implementation("org.json:json:20160212")
     implementation("org.glassfish.jaxb:jaxb-runtime:2.3.2")
+    implementation ("org.apache.pdfbox:pdfbox:1.8.7") {
+        // Provided by ZAP.
+        exclude(group = "commons-logging")
+        // Not needed.
+        exclude(group = "org.apache.pdfbox", module = "jempbox")
+    }
+    implementation ("org.apache.odftoolkit:simple-odf:0.7-incubating")
```

Comment:
These shouldn't be needed (at least for now).

Reply:
Just the odftoolkits?

Reply:
Yes.

Reply:
This can also be removed:
https://github.com/zaproxy/zap-extensions/pull/2177/files#diff-44ad9bafd7552572f2f087ebb2926897R13-R19